reference/JsGetModuleHostInfo.md

```diff
@@ -0,0 +1,20 @@
+Retrieve the host info for the specified module.
```

Comment:
Seems like content for `JsSetModuleHostInfo`

Reply:
This line was intentional, JsSetModuleHostInfo attaches information to the module; JsGetModuleHostInfo reads it back/retrieves it - maybe there's a better way of wording it?

Reply:
No i mean the entire file has content for `JsSetModuleHostInfo` instead of `JsGetModuleHostInfo` :)

Reply:
I re-chedked it was all correct other than the function name itself which I've now updated. The 2 functions are kinda similar...

Reply:
No worries. The added explanation in those reference files is awesome.